laravel/socialite/2015-07-23.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Insecure state generation
+link:      https://github.com/laravel/socialite/pull/91
```

Comment:
> There are no plans to document it sorry.

just having an information about the fact that there is some kind of security issue but no information about the issue details is not very helpful.
If you find a security issue in your framework or library you should have a way to inform your users and let them know how severe the issue is and how they can protect themselfs. This information is important so that users can decide on whether they are vulnerable or not and in which way.
It is also recommended to reserve a CVE number for relevant security issues: https://cve.mitre.org/cve/request_id.html Assignment is as easy as asking for one by writing to cve-assign@mitre.org. CVE provides a general way of dealing with security issues and keep track of the information on them.